src/routine.cpp

```diff
@@ -24,6 +24,7 @@ namespace clblast {
 // The constructor does all heavy work, errors are returned as exceptions
 Routine::Routine(Queue &queue, EventPointer event, const std::string &name,
                  const std::vector<std::string> &routines, const Precision precision,
+                 std::initializer_list<const char *> source_pct,
```

Comment:
What's the full name of `source_pct`? At first I read 'ptr', perhaps it is a good to idea to use to full name (if not too long of course)